docs/src/main/paradox/server-side/graceful-termination.md

```diff
@@ -31,15 +31,16 @@ This can be used to signal parts of the application that the HTTP server is shut
 Note also that for more advanced shut down scenarios you may want to use the @extref[Coordinated Shutdown](akka-docs:/actors.html#coordinated-shutdown) capabilities of Akka.
 
 Next, all in flight requests will be handled. If a request is "in-flight" (being handled by user code), it is given `hardDeadline` time to complete.
- 
-- if user code emits a response within the timeout, then this response is sent to the client
+
+- if connection has no "in-flight" request, connection is terminated immediately  
```

Comment:
Grammar: if a connection has no "in-flight" request, it is terminated immediately